src/main.ts

```diff
@@ -424,6 +428,15 @@ export default class CommentatorPlugin extends Plugin {
 		await this.saveSettings();
 	}
 
+	/**
+	 * Check if a file path is in an excluded folder
+	 */
+	isFileExcluded(filePath: string): boolean {
+		return this.settings.excluded_folders.some(folder =>
+			filePath.startsWith(folder)
```

Comment:
I am not sure whether the intention is for the `excluded_folders` setting paths to match (a part of) the absolute path, _any_ part, or only the folder name.